### PR TITLE
Alert Component

### PR DESCRIPTION
**Pull request by nicosammito**
Implementation of Alert component as described in #9 with just some adjustments made.

### CODE REVIEW

**Comment by nicosammito** [comment]
Still need to add the alert component without Body => therefore draft

File: src/components/alert/Alert.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    const {variant = "primary", dismissible = false, icon = true, title, onClose = (event) => {}, children} = props
+
+    return <div className={`alert alert--${variant}`}>
+        <div className={"alert__header"}>
+            <div className={"alert__header-wrapper"}>
+                {icon ? <AlertIcon variant={variant}/> : null}
+                <span className={"alert__heading"}>{title}</span>
+            </div>
+            {dismissible ? <span className={"alert__dismissible"} onClick={event => onClose(event)}><IconX/></span> : null}
```
Comment (Taucher2003) [inline src/components/alert/Alert.tsx:37]: Just an idea, but would this work?
```suggestion
            {dismissible ? <span className={"alert__dismissible"} onClick={onClose}><IconX/></span> : null}
```

**Review by Taucher2003 — COMMENTED** [review]
Looks good to me (not approving yet since you mentioned that some work still needs to be done)

File: src/components/alert/Alert.tsx (hunk 1 above)
Comment (nicosammito) [reply src/components/alert/Alert.tsx:37]: Hmm could be. I will try and come back to you

Comment (nicosammito) [reply src/components/alert/Alert.tsx]: > Just an idea, but would this work?

Yep, appreciate that 👍🏻

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**